frontend/src/store/objectStore.ts

```diff
@@ -62,8 +62,10 @@ export const useObjectStore = defineStore('objectStore', () => {
         // TODO: Tags
 
         // Populate object
-        obj.metadata = metadataResponse.data[0];
-        obj.name = obj.metadata.metadata.find((x: any) => x.key === 'name')?.value;
+        if (metadataResponse.data.length && metadataResponse.data[0]) {
```

Comment:
Order of checking should be inverted. Checking for existence should always take precedence over checking an attribute of said object in order to best take advantage of short circuiting.
```js
if (metadataResponse.data[0] && metadataResponse.data.length) {
```